### PR TITLE
Move some changeset to the "Removals" section of CHANGELOG

### PR DESCRIPTION
**Pull request by Amxx**
#### PR Checklist
- [x] Changeset entry (run `npx changeset add`)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,27 @@
 
 ### Removals
 
-The following contracts were removed:
+The following contracts, libraries and functions were removed:
 
+- `Address.isContract` (because of its ambiguous nature and potential for misuse)
+- `Checkpoint.History`
```
Comment (ernestognw) [inline CHANGELOG.md:8]:
```suggestion
- `Checkpoints.History`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,27 @@
...
 - `GovernorVotesComp`
+- `GovernorProposalThreshold` (deprecated since 4.4)
 - `PaymentSplitter`
-- `TokenTimelock` (removed in favor of `VestingWallet`)
+- `PullPayment`
+- `SafeMath`
+- `SignedSafeMath`
+- `Timers`
+- `TokenTimelock` (in favor of `VestingWallet`)
+- All escrow contracts (`Escrow`, `ConditionalEscrow`, `RefundEscrow`)
```
Comment (ernestognw) [inline CHANGELOG.md:22]:
```suggestion
- All escrow contracts (`Escrow`, `ConditionalEscrow` and `RefundEscrow`)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,26 @@
...
+- All crossChain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```
Comment (ernestognw) [inline CHANGELOG.md:23]:
```suggestion
- All CrossChain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```

**Review by ernestognw — CHANGES_REQUESTED** [review]
Minor nits

**Review by ernestognw — CHANGES_REQUESTED** [review]
Minor nits

**Comment by ernestognw** [comment]
On a side note, the Changesets automatically parse and inserts the corresponding PR to each Changelog entry. Is this something we may like to keep somehow? I think it's important historical information and we're removing that metadata in the removals section

**Comment by Amxx** [comment]
For 
- `ERC20Snapshot`
- `ERC20VotesComp`
- `GovernorVotesComp`
- `PaymentSplitter`
we did not include the PR link. I personnally don't think its essential here, but I'm open to discussion.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,13 +2,26 @@
...
+- `GovernorProposalThreshold` (deprecated since 4.4)
 - `PaymentSplitter`
-- `TokenTimelock` (removed in favor of `VestingWallet`)
+- `PullPayment`
+- `SafeMath`
+- `SignedSafeMath`
+- `Timers`
+- `TokenTimelock` (in favor of `VestingWallet`)
+- All escrow contracts (`Escrow`, `ConditionalEscrow` and `RefundEscrow`)
+- All CrossChain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```
Comment (Amxx) [inline CHANGELOG.md:23]: Should we do lowercase "crosschain" following the folder's name?
```suggestion
- All crosschain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```

**Comment by ernestognw** [comment]
> For 
> 
> 
> 
> - `ERC20Snapshot`
> 
> - `ERC20VotesComp`
> 
> - `GovernorVotesComp`
> 
> - `PaymentSplitter`
> 
> 
> 
> we did not include the PR link. I personnally don't think its essential here, but I'm open to discussion.

Yes, I also don't think it's essential but we might appreciate it in the future, especially looking for discussion about why we remove X or Y.

Still LGTM, just wanted to make sure we're all aware of that

**Review by ernestognw — DISMISSED** [review]
Thanks 😀

**Review by ernestognw — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (frangio) [reply CHANGELOG.md:24]:
```suggestion
- All cross-chain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```

**Comment by frangio** [comment]
Yeah I think we should include the PR links so that it's easier to trace where the change was introduced.

**Pull request merged**